.buildkite/blueoil-pipeline.yml

```diff
@@ -68,12 +68,15 @@ steps:
       python3 -m venv blueoil_inference
       source blueoil_inference/bin/activate
       pip3 install -U pip
+      pip3 uninstall -y enum34
```

Comment:
I am not sure why we need this line.
Do we need this line at this time?

Reply:
If this uninstall operation is temporary for miss much of requirements.txt between each CI of existing PRs, I think it is better to add a comment or create an issue to remove this line 👍

Reply:
thx, I will make an issue

Reply:
https://github.com/blue-oil/blueoil/issues/1143